pythonforandroid/recipes/cffi/__init__.py

```diff
@@ -7,8 +7,8 @@ class CffiRecipe(CompiledComponentsPythonRecipe):
     Extra system dependencies: autoconf, automake and libtool.
     """
     name = 'cffi'
-    version = '1.4.2'
-    url = 'https://pypi.python.org/packages/source/c/cffi/cffi-{version}.tar.gz'
+    version = '1.11.5'
+    url = 'https://files.pythonhosted.org/packages/e7/a7/4cd50e57cc6f436f1cc3a7e8fa700ff9b8b4d471620629074913e3735fb2/cffi-1.11.5.tar.gz'
```

Comment:
Please don't hardcode the version here, but use the `{version}` string.
So basically simply changing the default `version` should be enough since it also keeps the nice short url version without hat long hash.

Reply:
Hi @AndreMiras thanks for feedback, I actually thought that pypi changed its URLS and hash was included, so the `{version}` trick was not working anymore. But I've just tested, and it still work, so indeed it's better to keep former URL. I'll update the PR in the afternoon.

Reply:
@AndreMiras done